### ISSUE TITLE
The default value of parameter count_include_pad of torch.nn.Functional.avg_pool2d()

### ISSUE THREAD

**Issue by knsong** [pytorch/pytorch#9986]
## Issue description
Is the default value of parameter `count_include_pad ` of `torch.nn.Functional.avg_pool2d()`
 really `True` ?  
## Code example
`import numpy as np`
`import caffe`
`import torch.nn.Functional as F`
`input = ...`
`torch_output = F.avg_pool2d(input, kernel_size = 3, stride = 1, padding = 1, )`
`caffe_output = avg_pooling_in_caffe(input, kernel_size = 3, stride = 1, padding = 1)`

It's obvious that the average pooling in caffe is equal to `F.avg_pool2d()` with parameter `ceil_mode = True` and `count_include_pad = True` in pytorch. Right? But when I call`F.avg_pool2d()` with `count_input_pad` set as default, the result will be `torch_output  != caffe_output`, while `F.avg_pool2d()` with 'count_input_pad' set explicitly as `True`, the result will be `torch_output == caffe_output`. So is really the default value of parameter `count_include_pad ` of `torch.nn.Functional.avg_pool2d()` `True`  as the [doc](https://pytorch.org/docs/0.4.0/nn.html#torch.nn.functional.avg_pool2d) says?

## System Info
- PyTorch or Caffe2: PyTorch
- How you installed PyTorch (conda, pip, source): conda install pytorch torchvision cuda80 -c pytorch

- OS: centos
- PyTorch version: 0.4.0
- Python version: 2.7
- CUDA/cuDNN version: 8.0

**Comment by SsnL** [comment]
You are right. This is fixed in 0.4.1 :)

**Comment by knsong** [comment]
@SsnL  But shouldn't you update the [docs](https://pytorch.org/docs/0.4.0/nn.html#torch.nn.functional.avg_pool2d) for the version 0.4.0? It's misleading.

**Comment by soumith** [comment]
docs are updated now.
